cpuid/src/transformer/mod.rs

```diff
@@ -66,7 +69,23 @@ pub enum Error {
     VcpuCountOverflow,
     /// The max size has been exceeded
     SizeLimitExceeded,
+    /// A call to an internal helper method failed
+    InternalError(super::common::Error),
```

Comment:
nit: use super::common and then just write common::Error here.

Reply:
If I do this there will be conflicts between the Error defined here and common::Error. I will have to use self::Error instead of Error. I would keep it as it is